src/main/resources/resources/logisim/strings/circuit/circuit.properties

```diff
@@ -50,7 +50,6 @@ circuitLabelLocAttr = Label Location
 circuitName = Circuit Name
 circuitNamedBoxFixedSize = Use fixed box-size
 CircuitSameInputOutputLabel = Circuit name equals to a label of an input or an output. Please specify another circuit name.
-circuitVhdlPath = VHDL architecture file path
```

Comment:
Got problems seeing why these strings are removed?

Reply:
I removed an outdated and unused attribute that is replaced by the VHDL-component